src/3rdparty/mjcpy2/CMakeLists.txt

```diff
@@ -46,4 +47,4 @@ message("osg includes: ${OSG_INCLUDE_DIR}")
 include_directories(SYSTEM "${CMAKE_SOURCE_DIR}")
 include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/Boost.NumPy")
 boost_python_module(mjcpy mjcpy2.cpp  mujoco_osg_viewer.cpp)
-target_link_libraries(mjcpy "${CMAKE_SOURCE_DIR}/mjpro/libmujoco.so" ${OSG_LIBRARIES} boost_numpy)
+target_link_libraries(mjcpy "${CMAKE_SOURCE_DIR}/mjpro/bin/libmujoco131.so" ${OSG_LIBRARIES} boost_numpy)
```

Comment:
Something else to consider here is that the mjpro directory is now called mjpro131 on download, so we should either change this to be mjpro131/ or make this clear in the install instructions to rename the directory.